cloudify-widget-pool-manager/src/main/resources/pool-manager-app-context.xml

```diff
@@ -137,7 +137,25 @@
 
     <bean id="cloudNodesDao" class="cloudify.widget.pool.manager.NodeMappingsDao"/>
 
-    <bean id="pingAction" class="cloudify.widget.pool.manager.actions.PingAction"></bean>
+    <bean id="pingAction" class="cloudify.widget.pool.manager.actions.PingAction"/>
+
+    <bean id="cloudServerApiFactory" class="cloudify.widget.pool.manager.CloudServerApiFactory"/>
+
+    <bean id="softlayerRestApi" class="cloudify.widget.softlayer.SoftlayerRestApi"/>
+
+
+    <bean id="softlayerCatalogManager" class="cloudify.widget.softlayer.SoftlayerCatalogManager">
+        <property name="stalePeriod" value="3600000"/>
+        <property name="pricesJSONArrayTemplate" value="[{'id':34807},{'id':27023},{'id':32500},{'id':32627},{'id':23070},{'id':35310},{'id':33483}]"/>
```

Comment:
Those are the price IDs that jclouds added for us without us knowing, besides the hardwareIDs specified in the pool